vimeo-networking/src/main/java/com/vimeo/networking/model/uploadquota/Periodic.java

```diff
@@ -22,33 +22,60 @@
  * SOFTWARE.
  */
 
-package com.vimeo.networking.model;
+package com.vimeo.networking.model.uploadquota;
 
 import com.google.gson.annotations.SerializedName;
 import com.vimeo.stag.UseStag;
 
+import org.jetbrains.annotations.Nullable;
+
 import java.io.Serializable;
 
 /**
- * Created by kylevenn on 8/19/15.
+ * Represents the periodic quota limits imposed on the user. If there are no periodic limits imposed on the user, then
+ * the fields will all be {@code null}.
  */
-@SuppressWarnings("unused")
 @UseStag
-public class Quota implements Serializable {
+public class Periodic implements Serializable {
```

Comment:
(Optional)(Brainstorm)

Each one of these objects has a "free", "max", and "used". Could it be useful to have an interface on these objects so that we could have a system that doesn't care if it's a periodic object, a lifetime object, or a space object? I haven't put too much thought into this, but you might want to be able to use these objects interchangeably.

Reply:
I would be okay with creating a `Quota` or `UploadLimit` interface. The only concern I have is regarding the nullability of each field. `Periodic` and `Lifetime` both have nullable fields, but `Space` is non null.

Reply:
Well that sucks. Not really much you can do about that.
Well the fields could still be non null, but the interface methods are nullable. But then that's taking a hit to the getters of Space.

If it'll clean up the code downstream, then I'd say it's worth it. If it won't, then we can hold off. No need to over architect. If the use case calls for it and it'll look prettier if these three have an interface then I'd vote a common interface with nullable fields.

Reply:
I'll add it. I forgot that you can change the nullability of the implementation. So we have

```java
public interface Quota {

    @Nullable
    Long getFree();

    @Nullable
    Long getUsed();

    @Nullable
    Long getMax();

}
```
but the `Space` class implements non null versions of those, so users of `Space` don't need to handle nullability, but users of the `Quota` interface do, which is fair.

Reply:
Agreed, that's rad.